### ISSUE TITLE
Prototype search-results features which call attention to exorbitant proposed prices

### ISSUE THREAD

**Issue by ericronne** [18F/calc#2037]
Proposed prices which are more than one standard deviation above the average are not considered fair and reasonable. Therefore let's prototype a feature which highlights that fact to users.

### When the proposed price is within one standard deviation of the average …

![image](https://user-images.githubusercontent.com/9259185/42470184-53491c32-837f-11e8-80b8-1920d57c0656.png)

- [x] Color the proposed-price graph text (and its vertical rule) with #436A79 ($color-gray-dark)
- [ ] Summarize the difference between the average and proposed in parenthesis ($color-gray-dark), underneath the proposed text box

### When the proposed price is more than one standard deviation above the average …

![image](https://user-images.githubusercontent.com/9259185/42470559-78410904-8380-11e8-890e-b04853f38719.png)

- [ ] Color the proposed-price graph text (and its vertical rule) with #981B1E ($color-red-darkest)
- [ ] Summarize the difference between the average and proposed in parenthesis ($color-red-darkest), underneath the proposed text box
- [ ] Block out the red vertical rule behind the standard-dev number, in case they overlap (nice to have)

cc @KellyBailey

**Comment by ericronne** [comment]
Blocked by #2035
